nautobot/docs/release-notes/index.md

```diff
@@ -1,18 +1,14 @@
 # Release Notes
 
-## [2.4 - December 2024](version-2.4.md)
+## [2.4 - January 2025](version-2.4.md)
 
-- Apps Marketplace Page and Installed Apps Page Tile View
+- Virtual Device Context Data Models
```

Comment:
These entries were alphabetized in previous releases. Are we defining a new convention here, and if so, can we document what that convention is?

Reply:
It's me trying to order them based on importance/relevance to end users. So the "documented standard" probably is just editorial review of the PO.

Reply:
Ack, fair enough. :-)